packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -126,6 +127,41 @@ class CupertinoCheckbox extends StatefulWidget {
   /// Defaults to [CupertinoColors.activeBlue].
   final Color? activeColor;
```

Comment:
`activeColor` and `inactiveColor` should reference `fillColor`, so it is immediately obvious what relationships they have with each other.